src/collective/sitter/content/sitter.py

```diff
@@ -204,11 +221,14 @@ def get_experiences(self):
         objects = [x.to_object for x in sitter_folder.experiences]
         return [obj for obj in objects if obj.UID() in set(self.experiences)]
 
+    def get_details(self):
+        return self.details.output
+
     def abbreviated_details(self, length):
-        if not self.details:
+        if not self.get_details():
```

Comment:
Since computing the richtext output involves calling a transformer, it might be worth storing it in a variable first rather than calling 'get_details` twice.

Reply:
done